rpc_util.go

```diff
@@ -522,6 +522,42 @@ func (o MaxRetryRPCBufferSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRetryRPCBufferSizeCallOption) after(c *callInfo, attempt *csAttempt) {}
 
+// PerRPCUnaryClientInterceptor returns a CallOption that specifies an interceptor
+// for unary RPCs. Interceptors are invoked in order they are passed to unary RPCs.
+//
+// CallOption interceptors are invoked after DialOption interceptors.
+func PerRPCUnaryClientInterceptor(f UnaryClientInterceptor) CallOption {
+	return UnaryClientInterceptorCallOption{UnaryClientInterceptor: f}
+}
+
+// UnaryClientInterceptorCallOption is a CallOption that specifies an interceptor
+// for unary RPCs.
+type UnaryClientInterceptorCallOption struct {
```

Comment:
Same.  (Actually I think all these call option structs are experimental?)

And for the streaming versions, too.